tests/e2e/conftest.py

```diff
@@ -686,10 +687,15 @@ def _tmp_bucket_create(
 ) -> Iterator[Tuple[str, Helper]]:
     tmp_path = tmp_path_factory.mktemp("tmp_bucket")
     tmpbucketname = f"neuro_test_e2e_{uuid()}"
-    nmrc_path = _get_nmrc_path(tmp_path_factory, require_admin=False)
+    nmrc_path = _get_nmrc_path(tmp_path_factory, require_admin=True)
 
     helper = Helper(nmrc_path, tmp_path)
-    helper.create_bucket(tmpbucketname)
+    try:
+        helper.create_bucket(tmpbucketname)
+    except AuthorizationError:
+        pytest.skip("No permission to create bucket for user E2E_TOKEN")
+        yield "", helper
```

Comment:
`pytest.skip()` raises an exception internally, this line is never executed.

Reply:
Ohhh, is that so. Let me check if I can remove it then.